ca_gazebo/models/create_april_tags/model.sdf

```diff
@@ -0,0 +1,98 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='create_april_tags'>
+    <static>1</static>
+    
+    <link name='AprilTag1'>
+      <pose frame=''>9.0 9.0 -1.2 0 0 0</pose>
```

Comment:
I think this pose has to be defined when you include the model, not here.

Reply:
There are four tags in this file... is there a way for defining poses for multiple objects of a `.sdf` file inside a `.world` file?

Reply:
Yes, when you include them. See [here](http://sdformat.org/spec?ver=1.6&elem=world#include_pose).

Reply:
I'll resolve this conversation, as we talked about this in Slack.